plugins/discovery-ec2/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -27,4 +27,6 @@ grant {
 
   // ec2 client opens socket connections for discovery
   permission java.net.SocketPermission "*", "connect";
+
+  permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
```

Comment:
🤦‍♀️

@jpountz @tlrx Given this, please re-review. Note that we have a similar grant in repository-s3 already, which is why the build did not run into it there.